docs/guide/data_types/maps.rst

```diff
@@ -243,23 +246,25 @@ long as it recognizes the instrument. To see what colormaps are available::
 
     >>> import sunpy.cm
     >>> sunpy.cm.cmlist.keys()   # doctest: +NORMALIZE_WHITESPACE
-    ['sohoeit304', 'sdoaia211', 'sohoeit195', 'trace1600', 'sdoaia94',
-     'trace284', 'trace1216', 'sdoaia304', 'trace1700', 'yohkohsxtal',
-     'trace195', 'sdoaia335', 'sdoaia1600', 'traceWL', 'stereohi2',
-     'sdoaia193', 'stereohi1', 'rhessi', 'trace171', 'trace1550',
-     'sohoeit284', 'stereocor2', 'hmimag', 'stereocor1', 'sdoaia1700',
-     'yohkohsxtwh', 'sohoeit171', 'hinodexrt', 'sdoaia131', 'sdoaia171',
-     'hinodesotintensity', 'sdoaia4500', 'soholasco3', 'soholasco2']
+    dict_keys(['sdoaia94', 'sdoaia131', 'sdoaia171', 'sdoaia193', 'sdoaia211',
+    'sdoaia304', 'sdoaia335', 'sdoaia1600', 'sdoaia1700', 'sdoaia4500',
+    'sohoeit171', 'sohoeit195', 'sohoeit284', 'sohoeit304', 'soholasco2',
+    'soholasco3', 'stereocor1', 'stereocor2', 'stereohi1', 'stereohi2',
+    'rhessi', 'yohkohsxtal', 'yohkohsxtwh', 'hinodexrt', 'hinodesotintensity',
+    'trace171', 'trace195', 'trace284', 'trace1216', 'trace1550', 'trace1600',
+    'trace1700', 'traceWL', 'hmimag', 'irissji1330', 'irissji1400',
+    'irissji1600', 'irissji2796', 'irissji2832', 'irissji5000', 'irissjiFUV',
+    'irissjiNUV', 'irissjiSJI_NUV'])
 
 The SunPy colormaps are registered with matplotlib so you can grab them like
 you would any other colormap::
 
-    >>> import matplotlib.pyplot as plt   # doctest: +SKIP
+    >>> import matplotlib.pyplot as plt
     >>> import sunpy.cm
 
 You need to import sunpy.cm or sunpy.map for this to work::
 
-    >>> cmap = plt.get_cmap('sdoaia171')   # doctest: +SKIP
+    >>> cmap = plt.get_cmap('sdoaia171')
```

Comment:
Should you not skip this as it's a plot?

Reply:
it doesn't generate a plot, it just returns the cmap object.